account_move_reconcile_helper/tests/test_account_move_reconcile_helper.py

```diff
@@ -0,0 +1,114 @@
+    def setUp(self):
+        super(TestAccountMoveReconcileHelper, self).setUp()
+        self.AccountObj = self.env["account.account"]
+        self.AccountJournalObj = self.env["account.journal"]
+        self.AccountMoveObj = self.env["account.move"]
+        self.AccountMoveLineObj = self.env["account.move.line"]
+
+        self.account_type_recv = self.env.ref("account.data_account_type_receivable")
+        self.account_type_rev = self.env.ref("account.data_account_type_revenue")
+
+        self.account_recv = self.AccountObj.create(
+            {
+                "code": "MRH-RECVT",
+                "name": "Receivable (test)",
+                "reconcile": True,
+                "user_type_id": self.account_type_recv.id,
+            }
+        )
+        self.account_sale = self.AccountObj.create(
+            {
+                "code": "MRH-SALET",
+                "name": "Receivable (sale)",
+                "reconcile": True,
+                "user_type_id": self.account_type_rev.id,
+            }
+        )
+
+        self.sales_journal = self.AccountJournalObj.create(
+            {
+                "name": "Sales journal",
+                "code": "MRH-SAJT",
+                "type": "sale",
+                "default_credit_account_id": self.account_sale.id,
+                "default_debit_account_id": self.account_sale.id,
+            }
+        )
+
+    def create_account_move(self, amount, debit_account, credit_account):
+        return self.AccountMoveObj.create(
+            {
+                "journal_id": self.sales_journal.id,
+                "line_ids": [
+                    (
+                        0,
+                        0,
+                        {
+                            "name": "Receivable line",
+                            "account_id": debit_account.id,
+                            "debit": amount,
+                        },
+                    ),
+                    (
+                        0,
+                        0,
+                        {
+                            "name": "Sales line",
+                            "account_id": credit_account.id,
+                            "credit": amount,
+                        },
+                    ),
+                ],
+            }
+        )
+
+    def test_01_partial_reconcile(self):
+        base_move = self.create_account_move(5000, self.account_recv, self.account_sale)
+
+        move1 = self.create_account_move(1000, self.account_sale, self.account_recv)
+
+        move2 = self.create_account_move(1000, self.account_sale, self.account_recv)
+
+        lines = self.AccountMoveLineObj.search(
+            [
+                ("move_id", "in", [base_move.id, move1.id, move2.id]),
+                ("account_id", "=", self.account_recv.id),
+            ]
+        )
+
+        lines.reconcile()
+        # For v13, need to force compute
+        lines._compute_reconciled_lines()
```

Comment:
Although the test works now, I do have a question.
While it works just fine when playing with the system.
 But for test script, without forcing to _compute_reconciled_lines(), calling the `line.reconcile_line_ids` will not get the computed result.

* Is this the affect of the new super optimized ORM in v13? (I experience this kind of things many times).
* Doing the force compute in the test is right way to go?